app/forms/base_adjustment_form.rb

```diff
@@ -48,10 +48,14 @@ def changed_value(val1, val2)
   def linked
     {
       type: self.class::LINKED_CLASS::LINKED_TYPE,
-      id: selected_record.dig(*self.class::LINKED_CLASS::ID_FIELDS),
+      id: linked_id(selected_record),
```

Comment:
just double checking - is this related to the translation changes or is this general code cleanup?

Reply:
never mind - understood the code after reading other parts